PowerPointLabs/PowerPointLabs/ActionFramework/ShortcutsLab/AddCustomShape/AddCustomShapeImageHandler.cs

```diff
@@ -4,10 +4,10 @@
 using PowerPointLabs.ActionFramework.Common.Interface;
 using PowerPointLabs.TextCollection;
 
-namespace PowerPointLabs.ActionFramework.ShapesLab
+namespace PowerPointLabs.ActionFramework.ShortcutsLab
```

Comment:
Sorry just spotted this.
Is the namespace supposed to be ShortcutsLab or ShapesLab?

Reply:
Ohh... did it change to ShortcutsLab?

Reply:
It seems the other shortcuts are using `ShortcutsLab` namespace. I'm not sure which would be more appropriate.

Reply:
I believe `ShortcutsLab` will be more appropriate here given that it's a feature under ShortcutsLab